src/Components/MobileNav/MobileNav.js

```diff
@@ -0,0 +1,276 @@
+            }}
+          />
+        </Div>
+      ) : (
+        <>
+          <Div padding="5px 15px" flex>
+            <form
+              onSubmit={e => {
+                e.preventDefault();
+                addLog(e);
+              }}
+            >
+              <Input placeholder="찾으시는 취미가 있으신가요?" />
+            </form>
+            <Span
+              color="gray"
+              onClick={() => {
+                setSearch(false);
+              }}
+            >
+              취소
+            </Span>
+          </Div>
+          <Div>
+            <Div padding="20px 5px">
+              <Span bold>최근 검색어</Span>
+            </Div>
+            {searchLog.map(log => (
+              <Div key={log.id} padding="10px 5px" flex>
+                {log.word}
+                <XIcon
+                  onClick={() => {
+                    delLog(log);
+                  }}
+                />
+              </Div>
+            ))}
+          </Div>
+          <Div>
+            <Div padding="5px">
+              <Span bold>추천 검색어</Span>
+            </Div>
+            <Div flex>
+              {KEYWORDS.map((keyword, index) => (
+                <KeyWord key={index}>{keyword}</KeyWord>
+              ))}
+            </Div>
+          </Div>
+        </>
+      )}
+      <UnderBar>
+        <Tab
+          onClick={e => {
+            setCurrentTab(1);
+            setSearch(false);
+          }}
+          selected={currentTab === 1}
+        >
+          <Video />
+          <div>클래스</div>
+        </Tab>
+        <Tab
+          onClick={e => {
+            setCurrentTab(2);
+            setSearch(false);
+            window.scrollTo(0, 0);
+          }}
+          selected={currentTab === 2}
+        >
+          <ListIcon />
+          <div>카테고리</div>
+        </Tab>
+        <Tab
+          onClick={e => {
+            setCurrentTab(3);
+            setSearch(false);
+          }}
+          selected={currentTab === 3}
+        >
+          <Create />
+          <div>create</div>
+        </Tab>
+        <Tab
+          onClick={e => {
+            setCurrentTab(4);
+            setSearch(false);
+          }}
+          selected={currentTab === 4}
+        >
+          <Person />
+          <div>마이 페이지</div>
+        </Tab>
+      </UnderBar>
+      {currentTab === 2 && (
+        <Menu>
+          {categories.map(category => (
+            <Div>
+              <Div color="#b8b8b8" size="1rem" bold>
+                {category.name}
+              </Div>
+              <Line />
+              {category.sub_category.map(sub => (
+                <Div size="0.9rem">{sub.name}</Div>
+              ))}
+            </Div>
+          ))}
+        </Menu>
+      )}
+    </Container>
+  );
+}
+
+const KEYWORDS = [
+  '제테크',
+  '어쩌구',
+  '추천검색',
+  '저쩌구',
+  '무슨수업',
+  '무슨무슨수업',
+];
+
+const Container = styled.div`
+  display: none;
+  width: 100vw;
+
+  @media ${({ theme }) => theme.mobile} {
+    display: block;
+  }
+`;
```

Comment:
```suggestion
const Container = styled.div`
  @media ${({ theme }) => theme.web} {
      display: none;
    }
    
  @media ${({ theme }) => theme.mobile} {
    display: block;
  }
`;
```
1. width: 100vw는 불필요한 속성으로 보이네요!
2. 위 코드 처럼 web도 지정해서 사용해주세요!
3. 기존 코드와 보여질 때는 다른 점이 하나도 없습니다. 하지만 개발자도구를 열어서 보면, 모바일로 전환되는 시점에 none 위에 block이 덮어 씌워집니다
4. 하지만 해당 방법으로 바꾸면 덮어씌워지는 개념이 아닌 none과 block으로 명확하게 나눠지게 됩니다.
5. 위와 같은 방법으로 코드의 질을 향상시켜보시죠!